brainsets_pipelines/vollan_moser_alternating_2025/pipeline.py

```
@@ -0,0 +1,580 @@
+
+        # Each file is downloaded individually from the ebrains data-proxy API
+        fpath = self.raw_dir / manifest_item.fname
+        if fpath.exists() and not self.args.redownload:
+            return fpath
+
+        fpath.parent.mkdir(parents=True, exist_ok=True)
+        url = f"{BUCKET_URL}/sharing_v4/{manifest_item.fname}"
+
+        self.update_status("DOWNLOADING")
+        logging.info(f"Downloading {manifest_item.fname}")
+        with requests.get(url, stream=True) as response:
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)